yarn.lock

```diff
@@ -6376,7 +6381,7 @@ lodash.without@~4.4.0:
   resolved "https://registry.yarnpkg.com/lodash.without/-/lodash.without-4.4.0.tgz#3cd4574a00b67bae373a94b748772640507b7aac"
   integrity sha1-PNRXSgC2e643OpS3SHcmQFB7eqw=
 
-lodash@4.17.15, lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.4:
+lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.4:
```

Comment:
Are we still using `lodash` in production indirectly?

Reply:
![image](https://user-images.githubusercontent.com/1798109/84497760-ac686b00-acaf-11ea-8b4a-9e1831cb9858.png)

yes, indirect by `yawn-yaml`

Reply:
![image](https://user-images.githubusercontent.com/6311784/84501644-1f291480-acb7-11ea-919a-96bba5fcdd50.png)